src/Editor/GUI/Editors/SkeletonEditor.cpp

```diff
@@ -0,0 +1,55 @@
+#include "SkeletonEditor.hpp"
+#include <Engine/Animation/Skeleton.hpp>
+#include <Engine/Hymn.hpp>
+#include <imgui.h>
+#include <Engine/Util/FileSystem.hpp>
+#include "Util/AssetConverterSkeleton.hpp"
+
+using namespace GUI;
+
+SkeletonEditor::SkeletonEditor() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Member variable 'SkeletonEditor::name' is not initialized in the constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuninitMemberVar)